DEV-633: Docs collator (#550)
cloudposse/docs: content/docs/tutorials/atmos-getting-started.md

---
title: "Getting started with Atmos"
description: "Learn what Atmos is and how you can start using it with stacks to simplify your DevOps Automation tasks."
sidebar_position: 2
---

## Atmos

> Documentation for atmos got moved to a dedicated site [atmos.tools](https://atmos.tools) 

Reviewer: ```suggestion
Documentation for Atmos has moved to [atmos.tools](https://atmos.tools) 
```
